src/shared/api/api.ts

```diff
@@ -47,30 +55,46 @@ function _fetch({
   })
 }
 
-function prefillMethod(method) {
-  return (config) =>
+function prefillMethod(method: string) {
+  return (config: any) =>
```

Comment:
There's no type we can use here?

Reply:
Config isn't typed yet ... and until we can ... I'd rather this just be any for the time being ... even tho it hurts every bone in my body.

Reply:
Ok as long as you feel bad about it 😆

Reply:
Feeling absolutely sick to my stomach about it